src/main/scala/PrivateHome/data.scala

```diff
@@ -223,7 +225,7 @@ object data {
       select.from(device as m)
     }.map(rs => device(rs)).list().apply().foreach(d => {
       d.switchtype match {
-        case "mqtt" =>
+        case Mqtt =>
```

Comment:
All uppercase would look nicer, as it is an own type, right?

Reply:
Yea